### ISSUE TITLE
WCAGもくもく会4の訳出見直し

### ISSUE THREAD

**Issue by momdo** [waic/wcag20#88]
https://ca11y.connpass.com/event/75363/
2018/01/16(火) 19:15 〜
### 対象となる解説書は次の通り

- [ガイドライン 2.1 を理解する](https://waic.github.io/wcag20/Understanding/keyboard-operation.html)
- [達成基準 2.1.1 を理解する](https://waic.github.io/wcag20/Understanding/keyboard-operation-keyboard-operable.html)
- [達成基準 2.1.2 を理解する](https://waic.github.io/wcag20/Understanding/keyboard-operation-trapping.html)
- [達成基準 2.1.3 を理解する](https://waic.github.io/wcag20/Understanding/keyboard-operation-all-funcs.html)

### 関連する達成方法集は次の通り（Flash/Silverlightを除く）

- [G21](https://waic.github.io/wcag20/Techniques/G21.html)
- [G90](http://waic.github.io/wcag20/Techniques/G90.html)
- [G202](http://waic.github.io/wcag20/Techniques/G202.html)
- [H91](http://waic.github.io/wcag20/Techniques/H91.html)
- [SCR2](http://waic.github.io/wcag20/Techniques/SCR2.html)
- [SCR20](http://waic.github.io/wcag20/Techniques/SCR20.html)
- [SCR35](http://waic.github.io/wcag20/Techniques/SCR35.html)
- [PDF3](http://waic.github.io/wcag20/Techniques/PDF3.html)
- [PDF11](http://waic.github.io/wcag20/Techniques/PDF11.html)
- [PDF23](http://waic.github.io/wcag20/Techniques/PDF23.html)

**Comment by bakera** [comment]
https://waic.github.io/wcag20/Understanding/keyboard-operation.html
>Optimized speech input, optimized mouse/pointer input, etc., are also good. The key is to provide keyboard input and control as well.
>最適化された音声入力、最適化されたマウス/ポインタ入力なども適している。鍵となるのは、キーボード入力及び制御を同様に提供することである。

「適している」はやや違和感があります。「入力などがあっても良い」とか?
後半は「同様に」とは言っていないように思いました。「制御も提供する」とか「制御も同時に提供する」くらい?

**Comment by bakera** [comment]
https://waic.github.io/wcag20/Understanding/keyboard-operation.html
>This guideline uses the term "keyboard interface" to acknowledge that Web content should be controlled from keystrokes that may come from a keyboard, keyboard emulator, or other hardware or software that generates keyboard or text input.
>このガイドラインでは、キーボード、キーボードエミュレータ、又はキーボード入力もしくはテキスト入力を生成するその他のハードウェアもしくはソフトウェアに由来するキーストロークから、ウェブコンテンツが制御されることが望ましいことを認識するために、「キーボードインタフェース」という用語を使用する。

ここの acknowledge は「認識する」というより「認める」に近い意味だと思います。「キーボードインタフェース」という用語はリアルキーボードでないものも含んでいる (認めている)、ということを伝えたい文章のはずです。が、もとの英文がなかなか厳しくどう訳せば望ましいのかがなんとも難しいところ。2文に分ける?

**Comment by momdo** [comment]
@bakera
> etc., are also good. 

なども(好ましい|都合が良い|有効である)
とかどうでしょうか。

>to provide keyboard input and control as well

「キーボード入力だけでなく制御も提供すること」ですかねえ…もうなんか”as well as”みたいな訳語ですけど。

**Comment by bakera** [comment]
@momdo 
to provide keyboard input and control as well ですが、文脈からして (keyboard input and control) はひとかたまりのように思います。前の文で音声やマウスの話をしているので、「鍵となるのは、キーボードによる入力及び制御も同時に提供することである。」くらいかなぁと。

**Comment by bakera** [comment]
https://waic.github.io/wcag20/Understanding/keyboard-operation-keyboard-operable.html
>It is assumed that the design of user input features takes into account that operating system keyboard accessibility features may be in use. 
>利用者の入力機能の設計は、オペレーティングシステムのキーボードアクセシビリティ機能を使用する可能性があることを考慮していると想定される。
→利用者が入力を行う機能の設計においては、オペレーティングシステムのキーボードアクセシビリティ機能が利用されている可能性を考慮することが期待される。

>For example, modifier key locking may be turned on. 
>例えば、修飾キーがロックされているかもしれない。
→例えば、修飾キーをロックする機能が有効になっているかもしれない。

>Content continues to function in such an environment, not sending events that would collide with the modifier key lock to produce unexpected results. 
>そのような環境でコンテンツは引き続き機能し、予期しない結果を生じるために修飾キーロックと衝突するイベントは送信されない。
→そのような環境でもコンテンツは機能し、修飾キーロックと衝突して予期しない結果を生じるようなイベントを送信しない (ようにすべきである)。

**Comment by bakera** [comment]
https://waic.github.io/wcag20/Understanding/keyboard-operation-trapping.html
>A puzzle applet 
>パズルアプレット 

この例、パズルである必要ないですよね (単なる感想)。

**Comment by momdo** [comment]
修飾キーの訳註はあってもよいのかもしれません。

https://twitter.com/izuizu/status/953234069659136000
https://twitter.com/ta_ka_ko_/status/953228943645003776

**Comment by momdo** [comment]
修飾キーの説明はWikipediaよりこっちな方がいいかなー。
http://e-words.jp/w/%E4%BF%AE%E9%A3%BE%E3%82%AD%E3%83%BC.html

**Comment by momdo** [comment]
proposal:
修飾キーとは、別のキーと組み合わせて使用し、そのキーの意味を変えるものである。例えば、Shift キーがこれに当たる。<a href="http://e-words.jp/w/%E4%BF%AE%E9%A3%BE%E3%82%AD%E3%83%BC.html">修飾キーとは - IT用語辞典</a>も参照のこと。

これを https://github.com/waic/wcag20/issues/88#issuecomment-357934772 の段落の直後に訳註として挿入する。
